src/utils/mmbot_strong_types.hpp

```diff
@@ -64,6 +64,12 @@ namespace antara
             st::addable_with<double>
     >;
 
+    using st_maker = st::type<
```

Comment:
for boolean you are not forced to use strong type.

a simple using make = bool; is ok

Reply:
That's really strange to have a st::arithmetic on bool

Reply:
I'm surprised it even compiled! Fixed.